vllm/model_executor/layers/mamba/ops/ssd_combined.py

```diff
@@ -38,6 +38,8 @@ def _mamba_chunk_scan_combined_fwd(x,
                                    dt_softplus=False,
                                    dt_limit=(0.0, float("inf")),
                                    out=None):
+    assert isinstance(chunk_size, int) and chunk_size > 0 and (chunk_size & (chunk_size - 1)) == 0, \
+        "chunk_size must be integer power of 2"
```

Comment:
nit: It would be clearer to move `isinstance(x, int) and x > 0 and (x & (x - 1)) == 0,` into a helper function and use it here:

```suggestion
    assert is_pow_2(chunk_size), "chunk_size must be integer power of 2"
```

Reply:
fixed in most recent commit